features/step_definitions/select-language-steps.js

```diff
@@ -42,3 +46,12 @@ Then(/^I should have Japanese language set$/, async function () {
   });
   expect(result.value).to.equal('ja-JP');
 });
+
+Then(/^I should have Spanish language set$/, async function () {
+  const result = await this.client.executeAsync((done) => {
+    daedalus.stores.profile.getProfileLocaleRequest.execute()
+      .then(done)
+      .catch((error) => done(error));
+  });
+  expect(result.value).to.equal('es-ES');
+});
```

Comment:
Additions to this file are only needed if they are used in feature tests like https://github.com/input-output-hk/daedalus/blob/master/features/select-language.feature

The feature test I linked currently verifies it's possible to select the Japanese language and "set" it, given that you start with no language set. A possible improvement would be adding a test to demonstrate switching from having language A set to language B set via the selection UI works as expected.

Reply:
@jmitchell If I created this new test for Spanish, would it be correct to
 1. rename `select-language.feature` into `select-japanese-language.feature`
 2. have the new test called `select-spanish-language.feature`
?